plasmapy/formulary/parameters.py

```diff
@@ -131,10 +131,10 @@ def mass_density(
         equivalent).  If ``density`` is a mass density, then it will be passed
         through and returned without modification.
 
-    particle : `~plasmapy.particles.Particle`
+    particle : `~plasmapy.particles.particle_class.Particle`
```

Comment:
Doesn't the previous version resolve through `__init__.py`?

Reply:
The way we have it set up, it needs the full namespace to resolve.  I'm not actually sure why that is, but was what motivated me to set up the reST substitutions.  Thanks for the review!

Reply:
You need the full path path because, even though `Particle` is exposed to the `plasmapy.particles` namespace, only one stub file is created for `Particle` at the full path `plasmapy.particles.particle_class.Particle`.  The original behavior of `automodapi` created two stub files for each location, but our version does not.

If you look the `plasmapy.particles` documentation page, then you'll notice the `Particle` link is actually point to `plasmapy.particles.particle_class.Particles`.